mypy/binder.py

```diff
@@ -103,19 +109,21 @@ def _get(self, key: Key, index: int=-1) -> Type:
                 return self.frames[i][key]
         return None
 
-    def push(self, node: Node, typ: Type) -> None:
-        if not node.literal:
+    def put_if_bindable(self, expr: Expression, typ: Type) -> None:
```

Comment:
I think it adds no information only if you know exactly how the binder works. When I read this code I had no idea that some expressions can be simply ignored. I do not like long names, but shorter names feels misleading.